hbase/pom.xml

```diff
@@ -33,8 +33,6 @@
   <properties>
     <!--library versions-->
     <interpreter.name>hbase</interpreter.name>
-    <hbase.hbase.version>2.4.12</hbase.hbase.version>
```

Comment:
the HBase interpreter is based on shell command, neither HBase nor Hadoop deps are required

Reply:
Quite strange, but I agree.